core/whistle-1.0.0/src/wh_util.erl

```diff
@@ -108,6 +108,10 @@
 
 -export([node_name/0, node_hostname/0]).
 
+-export([anonymous_caller_id_name/0
```

Comment:
@jamesaimonetti Initially I had the same idea but since I need to call this function from ecallmgr and I felt that made it inappropriate for whistle_apps lib